runtime/runtime/src/actions.rs

```diff
@@ -1294,7 +1292,7 @@ mod tests {
             &mut action_result,
             account_id,
             &DeleteAccountAction { beneficiary_id: "bob".parse().unwrap() },
-            ProtocolFeature::DeleteActionRestriction.protocol_version(),
```

Comment:
Hm, we have a number of (integration) tests that verify that protocol upgrade works for that feature, but at the same time for some feature are the only feature test as well. Not an issue here, but definitely should keep an eye out on not deleting those tests entirely, only adapting.

Reply:
Yep, will try my best, but that's where reviews help as well. I can't claim to know how all tests work and would make my best judgement.